app/auth/profile/deleteprofile.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState, Fragment, useRef, useCallback, useMemo } from "react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { GestureHandlerRootView } from "react-native-gesture-handler";
+import { router } from "expo-router";
+import { z } from "zod";
+import BottomSheet, {
+  BottomSheetView,
+  BottomSheetModal,
+  BottomSheetModalProvider,
+} from "@gorhom/bottom-sheet";
+import { View, StyleSheet, Text, TouchableOpacity, Modal, Pressable, Button } from "react-native";
+import FormContainer from "../../../components/Forms/FormContainer";
+import FormHeader from "../../../components/Forms/FormHeader";
+import FormField from "../../../components/Forms/TextInput";
+import SubmitButton from "../../../components/Forms/SubmitButton";
+import useProfile from "../../../hooks/useProfile";
+import { useAuthentication } from "../../../providers/AuthenticationProvider";
+import { useToast } from "../../../providers/ToastProvider";
+import { colors, ScaleSize, ScaleSizeH, ScaleSizeW } from "../../../utils/SharedStyles";
+
```

Comment:
Du har en del unused imports.

Ydermere vil jeg selv forestrække brugen af noget andet end BottomSheetModal, da efter jeg har læst deres docs tænker jeg ikke den bør blive brugt i denne kontekst.
https://gorhom.dev/react-native-bottom-sheet/modal

Jeg prøvede selv med en Alert fra react og synes selv det vil give god mening her. 
![image](https://github.com/user-attachments/assets/b8883bfc-04d0-4a70-8198-94ede9ac132c)

Reply:
Min kommentar var mere rettet mod, at brugen af BottomSheetModal i denne kontekst nok ikke er den bedste løsning. Vi bør sikre, at vores kodebase kan fungere som et solidt referencepunkt for kommende semestre med hensyn til, hvad de bør anvende.

Jeg har desuden drøftet det med min gruppe, og vi er alle enige om, at den nuværende implementering af confirmation ikke ser bedre ud end en alert.
<img src="https://github.com/user-attachments/assets/99ee1a74-67a5-4cab-82f2-08f013878256" alt="Phone screenshot" height="500" />
Jeg har undersøgt hvordan det ser ud på andre apps samt default på både iOS og Android.
<div style="display: flex; gap: 10px;">
  <img src="https://github.com/user-attachments/assets/a3f1bcfc-a1bb-49d0-a11f-ab148df7f1e5" height="500" />
  <img src="https://github.com/user-attachments/assets/e5f61980-3ea7-4ff5-bd38-c28fd98bd053" height="500" />
  <img src="https://github.com/user-attachments/assets/d6995a6a-bcdf-4cdc-b178-a5284c5f5ebd" height="500" />
</div>
Jeg mener, at det giver mest mening at følge en standard, der sikrer den bedste brugeroplevelse. Dette kan nemt opnås med en simpel alert, da confirmations typisk følger samme format. Det vil også gøre koden mere læsbar og reducere antallet af linjer i den nuværende implementering.